human-viz-bridge/src/iCubEffortBridge.cpp

```diff
@@ -0,0 +1,330 @@
+#include "sensor_msgs_Temperature.h"
+#include "std_msgs_Header.h"
+
+#include <iDynTree/ModelIO/ModelLoader.h>
+#include <iDynTree/Model/Model.h>
+#include <iDynTree/Model/Traversal.h>
+#include <TickTime.h>
+#include <yarp/dev/ControlBoardInterfaces.h>
+#include <yarp/dev/ITorqueControl.h>
+#include <yarp/dev/PolyDriver.h>
+#include <yarp/os/BufferedPort.h>
+#include <yarp/os/LogStream.h>
+#include <yarp/os/Network.h>
+#include <yarp/os/Node.h>
+#include <yarp/os/Property.h>
+#include <yarp/os/Publisher.h>
+#include <yarp/os/RFModule.h>
+#include <yarp/os/Time.h>
+#include <yarp/sig/Vector.h>
+
+#include <iDynTree/Core/EigenHelpers.h>
+
+#include <algorithm>
+#include <iostream>
+#include <limits.h>
+#include <string>
+#include <vector>
+#include <cmath>
+
+using namespace std;
+using namespace yarp::os;
+using namespace yarp::sig;
+using namespace yarp::dev;
+
+//---------------------------------------------------------------------------
+//Utility functions for parsing INI file
+static bool parseStringListOption(const Value &option, vector<string> &parsedList);
+//---------------------------------------------------------------------------
+
+struct EffortPublisher {
+    string linkEffort;
+    vector<size_t> jointDoFs;
+    Publisher<sensor_msgs_Temperature> *publisher;
+    sensor_msgs_Temperature effortMsg;
+};
+
+inline TickTime normalizeSecNSec(double yarpTimeStamp)
+{
+    uint64_t time = (uint64_t) (yarpTimeStamp * 1000000000UL);
+    uint64_t nsec_part = (time % 1000000000UL);
+    uint64_t sec_part = (time / 1000000000UL);
+    TickTime ret;
+
+    if (sec_part > UINT_MAX)
+    {
+        yWarning() << "Timestamp exceeded the 64 bit representation, resetting it to 0";
+        sec_part = 0;
+    }
+
+    ret.sec  = sec_part;
+    ret.nsec = nsec_part;
+    return ret;
+}
+
+class iCubEffortPublisherModule : public RFModule 
+{
+    iDynTree::VectorDynSize robotTorques;
+    PolyDriver m_robot;
```

Comment:
you should be consistent in how you declare members.
I prefer the `m_` version anyway.